addons/sale/models/sale_order.py

```diff
@@ -1644,6 +1645,7 @@ def _has_to_be_signed(self):
             and not self.is_expired
             and self.require_signature
             and not self.signature
+            and not self._is_confirmation_amount_reached()
```

Comment:
Is this necessary? Whether the order requires a signature should be independent of the payments, the state check should be enough :thinking:

Reply:
Indeed, fixed.